go.mod

```diff
@@ -2,12 +2,14 @@ module reconciler.io/runtime
 
 go 1.23.0
 
+toolchain go1.24.1
+
```

Comment:
```suggestion
```

Reply:
@scothis I was wondering about this myself, but it's not a breaking change, or is it?